### ISSUE TITLE
set-style(mark: (end: ">")) fails with bezier curve

### ISSUE THREAD

**Issue by raphCode** [johannes-wolf/cetz#185]
```
#import"@preview/cetz:0.1.1"

#cetz.canvas(length:1cm, {
  import cetz.draw: *
  set-style(mark: (end: ">"))
  bezier((0,0), (2, 2), (1, 0))
})
```
error: `expected color or none, found auto`

Expected result:
![image](https://github.com/johannes-wolf/typst-canvas/assets/15750438/9f11a061-af2a-4f00-9078-19bbe544739e)

**Comment by johannes-wolf** [comment]
I have opened a PR to change the styling system. This would resolve this bug, too.
Please have a look here: #189

**Comment by johannes-wolf** [comment]
Fixed with #200
